htlcswitch/link.go

```diff
@@ -1524,17 +1524,15 @@ func (l *channelLink) htlcManager() {
 		case qReq := <-l.quiescenceReqs:
 			l.quiescer.InitStfu(qReq)
 
-			pendingOnLocal := l.channel.NumPendingUpdates(
-				lntypes.Local, lntypes.Local,
-			)
-			pendingOnRemote := l.channel.NumPendingUpdates(
-				lntypes.Local, lntypes.Remote,
-			)
-			if err := l.quiescer.SendOwedStfu(
-				pendingOnLocal + pendingOnRemote,
-			); err != nil {
-				l.stfuFailf("%s", err.Error())
-				qReq.Resolve(fn.Err[lntypes.ChannelParty](err))
+			if l.noDanglingUpdates(lntypes.Local) {
```

Comment:
Is this commit going to be squashed before merge?

Reply:
Yeah I'll squash it.

Reply:
Nvm the conflicts are ridiculous if I try and squash it.